selfdrive/controls/lib/longitudinal_planner.py

```diff
@@ -141,6 +141,22 @@ def update(self, sm):
       accel_limits = [ACCEL_MIN, ACCEL_MAX]
       accel_limits_turns = [ACCEL_MIN, ACCEL_MAX]
 
+    # override accel using Accel Controller
+    if self.accel_controller.is_enabled:
+      # get min, max from accel controller
+      min_limit, max_limit = self.accel_controller.get_accel_limits(v_ego, accel_limits)
+      #print(f"Accel limits from controller: min_limit={min_limit}, max_limit={max_limit}")
+    if self.mpc.mode == 'acc':
+      # VOACC car, just give it max min (-1.2) so I can brake harder
+      accel_limits = [A_CRUISE_MIN, max_limit] if self.CP.radarUnavailable else [min_limit, max_limit]
+      # recalculate limit turn according to the new min, max
+      steer_angle_without_offset = sm['carState'].steeringAngleDeg - sm['liveParameters'].angleOffsetDeg
+      accel_limits_turns = limit_accel_in_turns(v_ego, steer_angle_without_offset, accel_limits, self.CP)
+    else:
+      # blended, just give it max min (-3.5) and max from accel controller
+      accel_limits = [ACCEL_MIN, ACCEL_MAX]
+      accel_limits_turns = [ACCEL_MIN, ACCEL_MAX]
```

Comment:
Let's abstract this completely inside `sunnypilot/selfdrive/controls/lib/accel_personality/accel_controller.py` and `sunnypilot/selfdrive/controls/lib/accel_personality/accel_controller.py`, so it works similarly to how we access DEC's outputs from those modules: https://github.com/sunnypilot/sunnypilot/blob/f4f48ee55c6b526b4665b2a26b42442aeb9a6cf8/selfdrive/controls/lib/longitudinal_planner.py#L110-L113

https://github.com/sunnypilot/sunnypilot/blob/f4f48ee55c6b526b4665b2a26b42442aeb9a6cf8/sunnypilot/selfdrive/controls/lib/dec/dec.py#L376-L390